apps/dialtone-documentation/docs/components/emoji-text-wrapper.md

```diff
@@ -47,6 +57,77 @@ vueCode='
 '
 showHtmlWarning />
 
+### Variants
+
+<code-well-header>
+  <div class="d-stack16" ref="variantsExample">
+    <dt-emoji-text-wrapper>
+      <button class="d-btn d-btn--primary d-btn--danger">
+        Button with shortcode :cry: emoji
+      </button>
+    </dt-emoji-text-wrapper>
+    <dt-emoji-text-wrapper>
+      Text only with unicode 😃 emoji
+    </dt-emoji-text-wrapper>
+    <dt-emoji-text-wrapper>
+      <button class="d-btn d-btn--primary">
+        Button wrapper :smile:
+      </button>
+    </dt-emoji-text-wrapper>
+    <dt-emoji-text-wrapper size="800">
+      Bigger emoji size :smile:
+    </dt-emoji-text-wrapper>
+    <dt-emoji-text-wrapper size="300">
+      Smaller emoji size :smile:
+    </dt-emoji-text-wrapper>
+  </div>
+</code-well-header>
+
+<code-example-tabs
+:htmlCode="() => $refs.variantsExample"
+vueCode='
+<dt-emoji-text-wrapper>
+  <button class="d-btn d-btn--primary d-btn--danger">
+    Button with shortcode :cry: emoji
+  </button>
+</dt-emoji-text-wrapper>
+<dt-emoji-text-wrapper>
+  Text only with unicode 😃 emoji
+</dt-emoji-text-wrapper>
+<dt-emoji-text-wrapper>
+  <button class="d-btn d-btn--primary">
+    Button wrapper :smile:
+  </button>
+</dt-emoji-text-wrapper>
+<dt-emoji-text-wrapper size="800">
+  Bigger emoji size :smile:
+</dt-emoji-text-wrapper>
+<dt-emoji-text-wrapper size="300">
+  Smaller emoji size :smile:
+</dt-emoji-text-wrapper>
+'
+/>
+
 ## Vue API
 
 <component-vue-api component-name="emojitextwrapper" />
+
+## Usage
+
+### Import
+
+```javascript
+// Note you must import from @dialpad/dialtone-vue. The emoji module is not included in the main
+// @dialpad/dialtone-vue package due to it's large size. You may wish to consider dynamically importing
+// the emoji module in your application.
```

Comment:
This is no longer a true statement, emoji-text-wrapper is now part of dialtone-vue